carbon-design-system/carbon
# fix(icon-build-helpers): disable svgo plugins that cause rendering issues

Closes #5565

After investigating the causes of our icon rendering issues, it seems related to these two optimization plugins from SVGO. Disabling them causes our icons to render correctly in Sketch.

#### Changelog

**New**

**Changed**

- Disable two SVGO plugins used for optimization

**Removed**

**Testing / Reviewing**

- Setup the sketch plugin per the instructions in the README
- Run the sync command from the carbon elements plugin
- Verify that icons render as intended in the symbols page
